packages/fiori/src/TimelineItem.js

```diff
@@ -150,6 +173,19 @@ class TimelineItem extends UI5Element {
 			Link,
 		];
 	}
+	get classes() {
+		return {
+			indicator: {
+				"ui5-tli-indicator" : true,
+				"ui5-tli-indicator-short-line": this.isShortLine,
+			},
+			arrowOrientation: {
+				"ui5-tli-bubble-arrow" : true,
+				"ui5-tli-bubble-arrow--left": this.axisOrientation === "vertical",
```

Comment:
After introducing the new enum, checks like this (hard-coded string comparison) must not be done. Use the enum type

Reply:
Done